specification/storage/resource-manager/Microsoft.Storage/stable/2023-05-01/storageTaskAssignments.json

```diff
@@ -0,0 +1,842 @@
+          "type": "string",
+          "description": "Text that describes the purpose of the storage task assignment"
+        },
+        "executionContext": {
+          "$ref": "#/definitions/StorageTaskAssignmentExecutionContext",
+          "description": "The storage task assignment execution context"
+        },
+        "report": {
+          "$ref": "#/definitions/StorageTaskAssignmentReport",
+          "description": "The storage task assignment report"
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Represents the provisioning state of the storage task assignment.",
+          "enum": [
+            "ValidateSubscriptionQuotaBegin",
+            "ValidateSubscriptionQuotaEnd",
+            "Creating",
+            "Succeeded",
+            "Deleting",
+            "Canceled",
+            "Failed"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": false
+          }
+        },
+        "runStatus": {
+          "$ref": "./common.json#/definitions/StorageTaskReportProperties",
+          "description": "Run status of storage task assignment"
+        }
+      },
+      "required": [
+        "taskId",
+        "enabled",
+        "description",
+        "executionContext",
+        "report"
+      ],
+      "description": "Properties of the storage task assignment."
+    },
+    "StorageTaskAssignmentExecutionContext": {
+      "type": "object",
+      "properties": {
+        "target": {
+          "$ref": "#/definitions/ExecutionTarget",
+          "description": "Execution target of the storage task assignment"
+        },
+        "trigger": {
+          "$ref": "#/definitions/ExecutionTrigger",
+          "description": "Execution trigger of the storage task assignment"
+        },
+        "constraints": {
+          "$ref": "#/definitions/ExecutionConstraints",
+          "description": "Execution constraints of the storage task assignment"
+        }
+      },
+      "required": [
+        "target",
+        "trigger"
+      ],
+      "description": "Execution context of the storage task assignment."
+    },
+    "ExecutionTarget": {
+      "type": "object",
+      "properties": {
+        "prefix": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "List of target prefixes"
+        },
+        "excludePrefix": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "List of target prefixes to exclude"
+        },
+        "reservedPathPrefix": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "List of reserved path prefixes"
+        }
+      },
+      "required": [
+        "prefix"
+      ],
+      "description": "Target for storage task assignment execution"
```

Comment:
>"description": "Target for storage task assignment execution" [](http://example.com/codeflow?start=6&length=61)

What does that really mean? What is the world of possible targets??

Reply:
Tricia -please add more description here, I think below makes sense.

"Target helps provide filter parameters for the objects in the storage account and forms the execution context for the storage actions."

Reply:
Fixed